pom.xml

```diff
@@ -481,6 +482,11 @@
         <version>${google.flogger.version}</version>
         <scope>runtime</scope>
       </dependency>
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
```

Comment:
Do we need to include any additional dependencies in Maven Shade Plugin configuration in the `gcs/pom.xml` and `bigquery/pom.xml`?

Reply:
Added `com.google.storage.v2` to gcs and bigquery shade plugin deps

Reply:
I still can see not relocated `com/google/storage/v2/` classes in shaded jar.

Reply:
```
I see the classes present in the jar, am i missing something here ?

hadoop-connectors/gcs/target on  submodule [$] via ☕ v1.8.0 on ☁️ prsagar@google.com(us-east1)
❯ jar tvf gcs-connector-hadoop3-3.0.0-SNAPSHOT-shaded.jar | grep "com.google.storage.v2"
     0 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/
  1774 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$1.class
 10680 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc.class
  1869 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$3.class
  1551 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageBaseDescriptorSupplier.class
  5532 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageStub.class
   551 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageFileDescriptorSupplier.class
  1881 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$2.class
  3633 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$MethodHandlers.class
  5779 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageImplBase.class
  1520 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageMethodDescriptorSupplier.class
  3749 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageBlockingStub.class
  3710 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageFutureStub.class
 61069 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$Builder.class
 18707 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams.class
 17805 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData$Builder.class
  1801 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse$1.class
 18179 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption.class
 19407 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams$Builder.class
   236 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ServiceConstantsOrBuilder.class
  1856 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption$1.class
 17328 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse.class
 20791 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest.class
   425 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeamOrBuilder.class
 26888 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest.class
 18478 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse.class
  2455 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$WriteStatusCase.class
   406 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponseOrBuilder.class
 18588 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam$Builder.class
  1717 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData$1.class
  1217 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/PredefinedObjectAcl$1.class
 19083 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption$Builder.class
 16000 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Builder.class
   780 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControlOrBuilder.class
 16507 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange.class
   949 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$2.class
 17805 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums$Builder.class
 27214 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest.class
 44023 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object.class
  2320 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$DataCase.class
 36404 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$Builder.class
  1696 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange$1.class
 17148 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange$Builder.class
   714 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponseOrBuilder.class
 21170 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse.class
 18384 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner$Builder.class
 18197 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse$Builder.class
   964 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponseOrBuilder.class
 23730 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec$Builder.class
  1731 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest$1.class
 25631 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StorageProto.class
 26233 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl$Builder.class
 33571 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest$Builder.class
 20875 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec.class
  2459 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$FirstMessageCase.class
 26600 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest$Builder.class
 28392 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse$Builder.class
  1717 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec$1.class
 17627 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam.class
   911 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequestOrBuilder.class
  1738 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse$1.class
   542 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryptionOrBuilder.class
 17068 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams.class
 25440 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest$Builder.class
 15247 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants.class
   407 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/OwnerOrBuilder.class
   739 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponseOrBuilder.class
  1745 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams$1.class
  1773 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest$1.class
  1724 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$1.class
  1295 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Values$1.class
   383 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedDataOrBuilder.class
 17068 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData.class
   860 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequestOrBuilder.class
  1717 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums$1.class
   298 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRangeOrBuilder.class
  1745 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$1.class
 21699 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$Builder.class
 17411 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner.class
 21514 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$Builder.class
  1321 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$2.class
   914 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$2.class
   494 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParamsOrBuilder.class
  1794 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest$1.class
  1780 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$1.class
  1689 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam$1.class
   383 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksumsOrBuilder.class
  1738 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$1.class
  1654 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$1.class
  7532 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Values.class
 23097 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl.class
 17959 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams$Builder.class
  1745 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl$1.class
  1883 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$MetadataDefaultEntryHolder.class
 20505 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest.class
  2500 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$WriteStatusCase.class
  1476 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ReadObjectRequestOrBuilder.class
  3373 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectOrBuilder.class
  1787 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams$1.class
   789 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/WriteObjectSpecOrBuilder.class
   396 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParamsOrBuilder.class
  1912 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequestOrBuilder.class
  5524 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/PredefinedObjectAcl.class
 18668 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse.class
  1647 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner$1.class
 17069 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums.class
 ```

hadoop2 

```
❯ jar tvf gcs-connector-hadoop2-3.0.0-SNAPSHOT-shaded.jar | grep "com.google.storage.v2"
     0 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/
  1774 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$1.class
 10680 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc.class
  1869 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$3.class
  1551 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageBaseDescriptorSupplier.class
  5532 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageStub.class
   551 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageFileDescriptorSupplier.class
  1881 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$2.class
  3633 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$MethodHandlers.class
  5779 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageImplBase.class
  1520 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageMethodDescriptorSupplier.class
  3749 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageBlockingStub.class
  3710 Wed Aug 11 22:23:04 IST 2021 com/google/storage/v2/StorageGrpc$StorageFutureStub.class
 61069 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$Builder.class
 18707 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams.class
 17805 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData$Builder.class
  1801 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse$1.class
 18179 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption.class
 19407 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams$Builder.class
   236 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ServiceConstantsOrBuilder.class
  1856 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption$1.class
 17328 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse.class
 20791 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest.class
   425 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeamOrBuilder.class
 26888 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest.class
 18478 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse.class
  2455 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$WriteStatusCase.class
   406 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponseOrBuilder.class
 18588 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam$Builder.class
  1717 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData$1.class
  1217 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/PredefinedObjectAcl$1.class
 19083 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryption$Builder.class
 16000 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Builder.class
   780 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControlOrBuilder.class
 16507 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange.class
   949 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$2.class
 17805 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums$Builder.class
 27214 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest.class
 44023 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object.class
  2320 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$DataCase.class
 36404 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$Builder.class
  1696 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange$1.class
 17148 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRange$Builder.class
   714 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponseOrBuilder.class
 21170 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse.class
 18384 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner$Builder.class
 18197 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteResponse$Builder.class
   964 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponseOrBuilder.class
 23730 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec$Builder.class
  1731 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest$1.class
 25631 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StorageProto.class
 26233 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl$Builder.class
 33571 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectRequest$Builder.class
 20875 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec.class
  2459 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$FirstMessageCase.class
 26600 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest$Builder.class
 28392 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse$Builder.class
  1717 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectSpec$1.class
 17627 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam.class
   911 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequestOrBuilder.class
  1738 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ReadObjectResponse$1.class
   542 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$CustomerEncryptionOrBuilder.class
 17068 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams.class
 25440 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest$Builder.class
 15247 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants.class
   407 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/OwnerOrBuilder.class
   739 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponseOrBuilder.class
  1745 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams$1.class
  1773 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequest$1.class
  1724 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$1.class
  1295 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Values$1.class
   383 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedDataOrBuilder.class
 17068 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ChecksummedData.class
   860 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusRequestOrBuilder.class
  1717 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums$1.class
   298 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ContentRangeOrBuilder.class
  1745 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$1.class
 21699 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$Builder.class
 17411 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner.class
 21514 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$Builder.class
  1321 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$2.class
   914 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectResponse$2.class
   494 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParamsOrBuilder.class
  1794 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest$1.class
  1780 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$1.class
  1689 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ProjectTeam$1.class
   383 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksumsOrBuilder.class
  1738 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequest$1.class
  1654 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$1.class
  7532 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/ServiceConstants$Values.class
 23097 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl.class
 17959 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParams$Builder.class
  1745 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectAccessControl$1.class
  1883 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Object$MetadataDefaultEntryHolder.class
 20505 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/StartResumableWriteRequest.class
  2500 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse$WriteStatusCase.class
  1476 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ReadObjectRequestOrBuilder.class
  3373 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectOrBuilder.class
  1787 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonObjectRequestParams$1.class
   789 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/WriteObjectSpecOrBuilder.class
   396 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/CommonRequestParamsOrBuilder.class
  1912 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/WriteObjectRequestOrBuilder.class
  5524 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/PredefinedObjectAcl.class
 18668 Wed Aug 11 22:22:50 IST 2021 com/google/storage/v2/QueryWriteStatusResponse.class
  1647 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/Owner$1.class
 17069 Wed Aug 11 22:22:48 IST 2021 com/google/storage/v2/ObjectChecksums.class
```

Reply:
Moved to `repackaged` dir

```
hadoop-connectors/gcs/target on  submodule [$!] via ☕ v1.8.0 on ☁️ prsagar@google.com(us-east1)
❯ jar tvf gcs-connector-hadoop2-3.0.0-SNAPSHOT-shaded.jar | grep "com.google.storage.v2"
     0 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/
  2086 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$1.class
 12981 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc.class
  2220 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$3.class
  1707 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageBaseDescriptorSupplier.class
  6546 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageStub.class
   707 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageFileDescriptorSupplier.class
  2232 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$2.class
  4140 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$MethodHandlers.class
  6676 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageImplBase.class
  1676 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageMethodDescriptorSupplier.class
  4529 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageBlockingStub.class
  4334 Wed Aug 11 22:23:04 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageGrpc$StorageFutureStub.class
 66802 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$Builder.class
 19994 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonObjectRequestParams.class
 18936 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ChecksummedData$Builder.class
  2035 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteResponse$1.class
 19505 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$CustomerEncryption.class
 20577 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonObjectRequestParams$Builder.class
   275 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstantsOrBuilder.class
  2168 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$CustomerEncryption$1.class
 18576 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteResponse.class
 22975 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusRequest.class
   464 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ProjectTeamOrBuilder.class
 30554 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest.class
 20194 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponse.class
  2767 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponse$WriteStatusCase.class
   445 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteResponseOrBuilder.class
 19719 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ProjectTeam$Builder.class
  1951 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ChecksummedData$1.class
  1412 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/PredefinedObjectAcl$1.class
 20253 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$CustomerEncryption$Builder.class
 16975 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstants$Builder.class
   897 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectAccessControlOrBuilder.class
 17716 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ContentRange.class
  1144 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponse$2.class
 18936 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectChecksums$Builder.class
 29515 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectRequest.class
 47338 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object.class
  2632 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest$DataCase.class
 42254 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest$Builder.class
  1930 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ContentRange$1.class
 18201 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ContentRange$Builder.class
   948 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponseOrBuilder.class
 24212 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectResponse.class
 19515 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Owner$Builder.class
 19328 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteResponse$Builder.class
  1315 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectResponseOrBuilder.class
 26031 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectSpec$Builder.class
  1965 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectRequest$1.class
 25709 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StorageProto.class
 28261 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectAccessControl$Builder.class
 36730 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectRequest$Builder.class
 22786 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectSpec.class
  2771 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest$FirstMessageCase.class
 30266 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteRequest$Builder.class
 32955 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectResponse$Builder.class
  1951 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectSpec$1.class
 18875 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ProjectTeam.class
  1184 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteRequestOrBuilder.class
  1972 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectResponse$1.class
   620 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$CustomerEncryptionOrBuilder.class
 18316 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonRequestParams.class
 28365 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusRequest$Builder.class
 16456 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstants.class
   446 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/OwnerOrBuilder.class
   973 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponseOrBuilder.class
  1979 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonRequestParams$1.class
  2007 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusRequest$1.class
  1958 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstants$1.class
  1529 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstants$Values$1.class
   422 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ChecksummedDataOrBuilder.class
 18316 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ChecksummedData.class
  1055 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusRequestOrBuilder.class
  1951 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectChecksums$1.class
   337 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ContentRangeOrBuilder.class
  1979 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponse$1.class
 23727 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponse$Builder.class
 18659 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Owner.class
 23542 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponse$Builder.class
  1633 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest$2.class
  1109 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectResponse$2.class
   533 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonObjectRequestParamsOrBuilder.class
  2028 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteRequest$1.class
  2014 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponse$1.class
  1923 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ProjectTeam$1.class
   422 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectChecksumsOrBuilder.class
  1972 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequest$1.class
  1888 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$1.class
  8000 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ServiceConstants$Values.class
 24813 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectAccessControl.class
 19090 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonRequestParams$Builder.class
  1979 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectAccessControl$1.class
  2039 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Object$MetadataDefaultEntryHolder.class
 23079 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/StartResumableWriteRequest.class
  2812 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponse$WriteStatusCase.class
  1671 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ReadObjectRequestOrBuilder.class
  3919 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectOrBuilder.class
  2021 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonObjectRequestParams$1.class
   945 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectSpecOrBuilder.class
   435 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/CommonRequestParamsOrBuilder.class
  2536 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/WriteObjectRequestOrBuilder.class
  5992 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/PredefinedObjectAcl.class
 20384 Wed Aug 11 22:22:50 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/QueryWriteStatusResponse.class
  1881 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/Owner$1.class
 18317 Wed Aug 11 22:22:48 IST 2021 com/google/cloud/hadoop/repackaged/gcs/com/google/storage/v2/ObjectChecksums.class
```